triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -195,7 +196,12 @@ object Trigger extends StrictLogging {
     val heartbeat = compiler.unsafeCompile(
       ERecProj(expr.ty, Name.assertFromString("heartbeat"), expr.expr)
     )
-    val machine = Speedy.Machine.fromPureSExpr(compiledPackages, heartbeat)
+    val machine =
+      // TODO: https://github.com/digital-asset/daml/issues/12208
+      //  get ride of `newLoggingContext`
```

Comment:
```suggestion
      //  get rid of `newLoggingContext`
```
Apart from the typo: can you clarify what the plan is to get rid of it? Where are you going to get your logging context from? The ticket itself doesn't mention that.